layouts/_default/single.html

```diff
@@ -9,51 +9,32 @@
     {{ partial "content-single.html" . }}
   {{ end }}
 
-  {{ range .Resources.ByType "page" }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "items") }}
-      {{ partial "fragments/items.html" (dict "resources" $.Resources "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "item") }}
-      {{ partial "fragments/item.html" (dict "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "table") }}
-      {{ partial "fragments/table.html" (dict "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "embed") }}
-      {{ partial "fragments/embed.html" (dict "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "member") }}
-      {{ partial "fragments/member.html" (dict "resources" $.Resources "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "logos") }}
-      {{ partial "fragments/logos.html" (dict "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "contact") }}
-      {{ partial "fragments/contact.html" (dict "Site" .Site "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (in .Params.fragment "footer") }}
-      {{ partial "fragments/footer.html" (dict "menus" $.Site.Menus "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "copyright") }}
-      {{ partial "fragments/copyright.html" (dict "menus" $.Site.Menus "Site" .Site "self" . ) }}
-    {{ end }}
-
-    {{ if and (not .Params.disabled) (eq .Params.fragment "buttons") }}
-      {{ partial "fragments/buttons.html" (dict "self" . ) }}
+  {{/*
+    This code may not be understandable immediately. This is trying to find all
+    the resources (read fragments) for each page and all the global fragments.
+    When combining global and local fragments, local ones have more priority.
+    So if there is only one footer fragment and that is global, that footer
+    fragment is shown. The same is true if there are no global footer fragments
+    and one local one. But if there is one global footer fragment and one local
+    footer fragment, the local one is rendered.
+    */}}
```

Comment:
Thanks for documenting. Would love some thoughts from @lauragift21.
I have the design in my head so it seems fine. I would probably clarify the difference between "fragment" and the custom fragment name aka filename. As the filename would be the factor in overwriting not the "fragment" type/name.

Reply:
Personally, I think going with the global fragment method is way better. It simplifies the layout and is a lot more reusable in terms of adding another "fragment", that brings me to ask do we still need to add the split layout using this method or everything is just going to be within a single layout?

Reply:
Alright. this explains it better.